config/orchestrations/codeready-workspaces/0.1/codeready-workspaces-cr.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  # The name prefixed with the kabanero instance name for multiple instance creation.
+  name: {{ .kabaneroInstanceName }}-codeready-workspaces
```

Comment:
We can't do this, because there is a name length limit in Kubernetes.  Having said that, there can only be a single `Kabanero` CR instance in a namespace, so I don't think we really need to pre-pend the name with the Kabanero instance name.